crud.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(db: Session):
+    return db.query(models.Author).all()
```

Comment:
The `get_all_authors` function does not implement pagination, which is required according to the task description. Consider adding parameters for pagination, such as `skip` and `limit`, and applying them to the query.